src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -35,6 +36,19 @@ object ProjectBuilder {
         (File(s"${name.toLowerCase}", "Dockerfile"), dockerfileContents)
     }
 
-    Project(databaseCreationScripts ++ dockerfiles)
+    val kubeFiles = KubernetesGenerator.generate(
+      KubernetesBuilder
+        .createServiceKubeFiles(
+          templefile.projectName,
+          templefile.services
+            .zip(1024 until Int.MaxValue)
+            .map {
+              case ((name, service), port) => (name, service, port)
+            }
```

Comment:
In fact, having this defined on the Templefile might be useful, to avoid reuse of the magic number 1024
```scala
def servicesWithPorts: Iterable[(String, ServiceBlock, Int)] =
  templefile.services
    .zip(1024 until Int.MaxValue)
    .map { case ((name, service), port) => (name, service, port) }
```

Reply:
Also wondering about `Iterator from 1024`/`Iterator.from(1024)` instead of explicitly stating the max value